mandatory/2-function-creation.js

```diff
@@ -15,26 +21,53 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice > 200) {
+    return (totalPrice - (totalPrice/100) * 10)
+  }
+  else {return (totalPrice - (totalPrice/100) * 5)} 
+
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+  let number = 0;
+  while (number < limit) {
+    number = number + 1;
+    if (number%2 === 0) continue;
+    console.log(number);  
+  }
+}
 
 /*
   Complete the buyTwoGetTheCheapestFree function: if user buys two items, the cheapest item will be free!
   The function should return the price to be paid once the discount is applied.
 */
-function buyTwoGetTheCheapestFree(price1, price2) {}
```

Comment:
This looks almost perfect.
Try to keep an eye on your indentation and spacing in the code. Even though it seems like it doesn't matter - it can make it much easier for other developers to read and understand your code 😄